src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Sdk.DefaultItems.props

```diff
@@ -28,7 +28,7 @@ Copyright (c) .NET Foundation. All rights reserved.
   </ItemGroup>
   <ItemGroup Condition=" '$(EnableDefaultItems)' == 'true' And '$(EnableDefaultNoneItems)' == 'true' ">
     <None Include="**/*" Exclude="$(DefaultItemExcludes);$(DefaultExcludesInProjectFolder)" />
-    <None Remove="**/*$(DefaultLanguageSourceExtension)" Condition=" '$(EnableDefaultCompileItems)' == 'true' "/>
+    <None Remove="**/*$(DefaultLanguageSourceExtension)" />
     <None Remove="**/*.resx" Condition=" '$(EnableDefaultEmbeddedResourceItems)' == 'true' "/>
```

Comment:
Should we also be excluding resx from None regardless of EnableDefaultEmbeddedResourceItems. Seems like that would be more consistent.

Reply:
Good catch, I think we should also exclude .resx.